vsts/vsts.yaml

```diff
@@ -594,7 +587,7 @@ jobs:
         # TODO #181 Config issue: must run on Debug builds only with valid PDBs.
         enabled: false
 
-      - task: securedevelopmentteam.vss-secure-development-tools.build-task-credscan.CredScan@2
+      - task: securedevelopmentteam.vss-secure-development-tools.build-task-credscan.CredScan@3
```

Comment:
##[warning]* Please upgrade the version of this build task in your build to major version: 3. For more information, please see: https://aka.ms/gdn-v1-deprecation